ceph-salt-formula/salt/ceph-salt/files/registries.conf.j2

```diff
@@ -0,0 +1,29 @@
+# For more information on this configuration file, see containers-registries.conf(5).
+#
+# Registries to search for images that are not fully-qualified.
+# i.e. foobar.com/my_image:latest vs my_image:latest
+[registries.search]
+registries = ["docker.io"]
+
+{% set insecure_registries = pillar['ceph-salt'].get('container', {}).get('insecure_registries', []) %}
```

Comment:
Could we model the data structure in a way where it works also with version 2 file format? With version 2, you would want to create something like:
```
unqualified-search-registries = ["docker.io"]

[[registry]]
prefix = "registry.suse.de"
#location = "registry.suse.de"
location = "10.86.4.8"
insecure = true
#blocked = false
```

So maybe having a list of dicts with registry name and insecure flag?

Reply:
> Could we model the data structure in a way where it works also with version 2 file format? 

Thanks for the feedback. For now, I'll keep that simple. This is just a short term solution, as I mentioned in previous comments, I think the better approach for AIRGAP solution is to

* Setup a local registry
* Put it behind a reverse proxy with SSL on
* Setup authentication 
* Accept the self-gen certs on all minions
* Login to the registry on all minions

Reply:
This is just a different configuration format for the client side. And V2 can do more things than V1. That's why I would like to see V2 used.